pkg/api/server/rest/authentication/token/tokenfile/tokenfile.go

```diff
@@ -0,0 +1,62 @@
+package tokenfile
```

Comment:
can we move this to `pkg/api/server/rest/authentication/authenticator/token/tokenfile`?